core/client/fs/src/main/java/alluxio/client/file/DoraCacheFileSystem.java

```diff
@@ -71,10 +71,12 @@ public class DoraCacheFileSystem extends DelegatingFileSystem {
   private final FileSystemContext mFsContext;
   private final boolean mMetadataCacheEnabled;
 
+  private static final long DEFAULT_MOCK_BLOCK_SIZE = 33554432L;
```

Comment:
For the sake of convenience of testing, we can reuse the block size config `alluxio.user.block.size.bytes.default`, so that we can compare the difference made by the block size across several runs.

Reply:
Good idea.